static/src/js/reducers/timeline.js

```diff
@@ -53,6 +55,12 @@ const timelineReducer = (state = initialState, action) => {
         }
       }
     }
+    case TOGGLE_TIMELINE: {
```

Comment:
We would prefer this action (and timeline isOpen state) be moved into the UI reducer. Look at how the `keyboardShortcutsModal` is stored there

```
timeline: {
  isOpen: true
}
```